smap_l2_gridder/crs.py

```diff
@@ -45,14 +45,14 @@ def col_row_to_xy(self, col: int, row: int) -> tuple[np.float64, np.float64]:
 # files CRS metadata may not match the authoritative value because of the
 # different varieties of WKT. That said, the CRS created by pyproj is the same.
 # i.e.
-# pyproj.crs.CRS.from_wkt(epsg_6933_wkt).to_wkt() != epsg_6933_wkt
+# pyproj.crs.CRS.from_wkt(EPSG_6933_WKT).to_wkt() != EPSG_6933_WKT
 # but
-# pyproj.crs.CRS.from_wkt(pyproj.crs.CRS.from_wkt(epsg_6933_wkt).to_wkt())
-#   == pyproj.crs.CRS.from_wkt(epsg_6933_wkt)
+# pyproj.crs.CRS.from_wkt(pyproj.crs.CRS.from_wkt(EPSG_6933_WKT).to_wkt())
+#   == pyproj.crs.CRS.from_wkt(EPSG_6933_WKT)
 
 # NSIDC EASE-Grid 2.0 Global CRS definition
 # from: https://epsg.org/crs/wkt/id/6933
-epsg_6933_wkt = (
+EPSG_6933_WKT = (
```

Comment:
<img width="858" alt="Screenshot 2024-11-27 at 2 16 14 PM" src="https://github.com/user-attachments/assets/77c36658-c161-469d-b9cc-ef87b29ddf14">

So it could actually be just the EPSG Code. The main reason I included the epsg.org full text is because that is canonical and the roundtrip is different when you write it to the file via crs.to_wkt()

It would probably clean it up a bit. 🤔

Reply:
And on the file I am putting the crs.to_wkt() flavor of wkt, not the canonical version.  so this might just make less clutter by dropping it.

Reply:
But I spaced doing this in this PR so I might slip it into the next one.

Reply:
f6a4c4a